frontend/front/src/pages/Public/Layout/Navbar.js

```diff
@@ -120,16 +124,16 @@ function Navbar(props) {
         TransitionComponent={Fade}
       >
         <Box>
-          {Object.keys(navbarItems[item]).map((subItem, index) => (
+          {Object.keys(navbarItems.LearnMore.items).map((subItem, index) => (
             <MenuItem
-              key={subItem}
+              key={index}
```

Comment:
It is not advised to use index as a key in React ([from the docs](https://react.dev/learn/rendering-lists#why-does-react-need-keys)); I would revert this to key={subItem} since each one is unique.

Reply:
fixed it.